src/main/java/at/hannibal2/skyhanni/features/mining/OreBlock.kt

```diff
@@ -44,7 +47,7 @@ enum class OreBlock(
     GOLD_ORE(Blocks.gold_ore, { inDwarvenMines || inCrystalHollows }),
     LAPIS_ORE(Blocks.lapis_ore, { inDwarvenMines || inCrystalHollows }),
     REDSTONE_ORE(
-        { it.block.equalsOneOf(Blocks.redstone_ore, Blocks.lit_redstone_ore) },
+        { it.block.equalsOneOf(mutableListOf<Block>().addRedstoneOres()) },
```

Comment:
please cache this list, instead of recreating it for every block we check against (its a lot, this is a huge diff)